### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by svenvanderburg**
* Update notebooks in  /examples
  - Remove test_fairworkflows
  - Rename noodles_fw to fairworkflows-quick-start and adapt it slightly so it is more represented as the quick start notebook for the project. 
* Update README to the current state of the project
* Add basic sphinx documentation, basically a copy of the readme + an API reference.

When this is merged I will setup readthedocs and add a link to the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,17 @@
 [![Coverage Status](https://coveralls.io/repos/github/fair-workflows/fairworkflows/badge.svg?branch=main)](https://coveralls.io/github/fair-workflows/fairworkflows?branch=main)
 [![PyPI version](https://badge.fury.io/py/fairworkflows.svg)](https://badge.fury.io/py/fairworkflows)
 [![fair-software.eu](https://img.shields.io/badge/fair--software.eu-%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8B-yellow)](https://fair-software.eu)
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/fair-workflows/fairworkflows/HEAD?filepath=examples%2Ftest_fairworkflows.ipynb)
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/fair-workflows/fairworkflows/main?filepath=examples%2Ffairworkflows-quick-start.ipynb)
```
Comment (svenvanderburg) [inline README.md:5]: This won't work yet, but should point to examples/fairworkflows-quick-start.ipynb once this branch is merged into main.

File: docs/index.rst [hunk 2]
```diff
@@ -0,0 +1,174 @@
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+   :caption: API Reference
+
+   reference/fairstep
+   reference/fairworkflow
+   reference/prov
```
Comment (svenvanderburg) [inline docs/index.rst:8]: I think this is all the user needs, do you agree?

File: docs/index.rst [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+`fairworkflows` python library
```
Comment (svenvanderburg) [inline docs/index.rst:10]: This is exactly the same text as in README.md but than converted to .rst using https://cloudconvert.com/md-to-rst , so no need to read both.

File: docs/index.rst (hunk 2 above)
Comment (raar1) [reply docs/index.rst:8]: Yes, I think so. Between this and the introductory notebook it essentially covers everything.

File: README.md [hunk 4]
```diff
@@ -18,96 +24,76 @@ The goals of the project are:
...
+### Example notebook
+* See [examples/fairworkflows-quick-start.ipynb](examples/fairworkflows-quick-start.ipynb) for a current example of using the fairworkflows library to build a workflow using plex rdf
 
 ## How is the ```fairworkflows``` library expected to be used?
-While this library could be used as a standalone tool to build/publish RDF workflows, it is intended more as a component to be used in a variety of other tools that seek to add FAIR elements to workflows. At present the library is used in the following tools:
+While this library could be used as a standalone tool to build/publish RDF workflows,
+it is intended more as a component to be used in a variety of other tools that seek to add FAIR elements to workflows. At present the library is used in the following tools:
 
 * [NanopubJL](https://github.com/fair-workflows/NanopubJL): A Jupyter Lab extension that adds a widget for searching the nanopublication servers, and helps the user fetch desired nanopubs through injection of the necessary python code into a notebook cell.
```
Comment (raar1) [inline README.md:98]:
```suggestion
```

File: docs/index.rst [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+
+How is the ``fairworkflows`` library expected to be used?
+---------------------------------------------------------
+
+While this library could be used as a standalone tool to build/publish
+RDF workflows, it is intended more as a component to be used in a
+variety of other tools that seek to add FAIR elements to workflows. At
+present the library is used in the following tools:
+
+-  `NanopubJL <https://github.com/fair-workflows/NanopubJL>`__: A
```
Comment (raar1) [inline docs/index.rst:148]:
```suggestion
```

File: docs/index.rst [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+   Jupyter Lab extension that adds a widget for searching the
```
Comment (raar1) [inline docs/index.rst:149]:
```suggestion
```

File: docs/index.rst [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+   nanopublication servers, and helps the user fetch desired nanopubs
```
Comment (raar1) [inline docs/index.rst:150]:
```suggestion
```

File: docs/index.rst [hunk 8]
```diff
@@ -0,0 +1,174 @@
...
+   through injection of the necessary python code into a notebook cell.
```
Comment (raar1) [inline docs/index.rst:151]:
```suggestion
```

**Review by raar1 — APPROVED** [review]
Great work Sven! The README update is great, and together with the example notebook and API reference this should be sufficient documentation for now.

**Pull request merged**